codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -87,16 +91,30 @@ class FileCodesignVisitor {
 </plist>
 ''';
 
-  void _initialize() {
-    entitlementsFile = tempDir.childFile('Entitlements.plist')..writeAsStringSync(_entitlementsFileContents);
-    remoteDownloadsDir = tempDir.childDirectory('downloads')..createSync();
-    codesignedZipsDir = tempDir.childDirectory('codesigned_zips')..createSync();
-  }
+  static const String fixItInstructions = '''
+Codesign test failed.
+
+We compared binary files in engine artifacts with those listed in
+entilement.txt and withoutEntitlements.txt, and the binary files do not match.
+*entitlements.txt is the configuartion file encoded in engine artifact zip,
+built by BUILD.gn and Ninja, to detail the list of entitlement files.
+Either an expected file was not found in *entitlements.txt, or an unexpected
+file was found in entitlements.txt.
+
+This usually happens during an engine roll.
+If this is a valid change, then BUILD.gn needs to be changed.
+Binaries that will run on a macOS host require entitlements, and
+binaries that run on an iOS device must NOT have entitlements.
+For example, If this is a new binarythat runs on macOS host, add it
```

Comment:
```suggestion
For example, if this is a new binary that runs on a macOS host, add it
```

Reply:
Very true, I really need to improve on writing. Thanks for correcting them Christopher!